scripts/tokenomics-emulation/event-handlers.ts

```diff
@@ -0,0 +1,43 @@
+import configFile from "./config.json";
+import fs from "fs";
+
+class EventHandlers {
+
+
+  /**
+   * Process the events to set smart contract address
+   * @param events Events
+   */
+   public static async handleEventsForSmartContractAddress(events, param) {
+     console.log("Handling Events");
```

Comment:
Add more specific log so it is clear which params you are handling

Reply:
Added